provisioning/miniprov/hare_mp/main.py

```diff
@@ -254,21 +255,202 @@ def test(args):
+        logging.error('Cluster failed to stop %s', resp[1])
+        raise Exception(
+            f'Command {cluster_stop} exited with error code '
+            f'{int(resp[2])}. Command output: {resp[1]}')
+    else:
+        logging.info('Cluster Stopped : %s', resp[0])
+        for line in resp[0]:
+            assert 'Cluster stop is in progress' not in line, \
+                'Cluster is in progress.' if 'Cluster is in progress.' \
+                else line
+
+
+def test_hare_postreq(cdf_file: str, timeinfo: str, logfile: str):
+    """Test suite is for restoring the setting after test."""
+    pcs_status = ['pcs', 'status']
+    cluster_start = ['cortx', 'cluster', 'start']
+    hctl_status = ['hctl', 'status', '-d']
+
+    logging.info('Start the Cluster')
+    resp = executecmds(cluster_start)
+    logging.info('cluster status: %s', resp[0])
+    if int(resp[2]):
+        logging.error('Cluster failed to start %s', resp[1])
+        resp = executecmds(['journalctl', '--since', timeinfo, '>', logfile])
+        logging.info('created journal log %s', logfile)
+        raise Exception(
+            f'Command {cluster_start} exited with error code {int(resp[2])}'
+            f'Command output: {resp[1]}')
+    else:
+        for line in resp[0]:
+            assert 'Cluster start operation performed' not in line, \
+                'Cluster not yet started.' if 'Cluster not yet started.' \
+                else line
+
+    cluster_sts = check_cluster_status(cdf_file)
+    if cluster_sts:
+        logging.error('Cluster status reports failure')
+        resp = executecmds(['journalctl', '--since', timeinfo, '>', logfile])
+        logging.info('created journal log.%s', logfile)
+        exit(-1)
+
+    logging.info('PCS: Check all services are up.')
+    sleep(20)
+    resp = executecmds(pcs_status)
+    logging.info('PCS status: %s', resp[0])
+    if int(resp[2]):
+        logging.error('PCS failed to updated the status %s', resp[1])
+    else:
+        for line in resp[0]:
+            assert 'stopped' not in line, 'Some services are not up.' \
+                if 'Some services are not up.' else line
+
+    logging.info('hctl: Check that all the services are up.')
+    sleep(10)
+    resp = executecmds(hctl_status)
+    logging.info('hctl status: %s', resp[0])
+    if int(resp[2]):
+        logging.error('hctl failed to updated the status %s', resp[1])
+        resp = executecmds(['journalctl', '--since', timeinfo, '>', logfile])
+        logging.info('created journal log.%s', logfile)
+        raise Exception(
+            f'Command {hctl_status} exited with error code {int(resp[2])}.'
+            f'Command output: {resp[1]}')
+    else:
+        for line in resp[0]:
+            assert 'stopped' not in line, 'Some services are not up.' \
+                if 'Some services are not up.' else line
+    logging.info('Successfully performed cleanup after testing')
+
+
+# @pytest.mark.sanity
```

Comment:
Do you need this decorator? If not, please remove the line. Otherwise please uncomment it,

Reply:
For now I will remove the decorator, once PR#1679 merged will move completed code accordingly.